StyleSheets/StyleSheets/StackViewStyleSheet.swift

```diff
@@ -1,31 +1,36 @@
 import UIKit
 
-open class StackViewStyleSheet: ViewStyleSheet<UIStackView> {
+open class StackViewStyleSheet<View: UIStackView>: ViewStyleSheet<View> {
```

Comment:
I feel like we would need to add generic attribute to the all of the stylesheets for consumers to be able to subclass. As people can subclass views but if they subclass stylesheet they will not be able to provide their view.

Let's say I create subclass `MyLabel: UILabel` then I want to have `MyLabelStyleSheet: LabelStyleSheet<MyLabel>` so then in `apply(to view: MyLabel)` but now I can not do that.